app/src/main/java/org/simple/clinic/analytics/Analytics.kt

```diff
@@ -131,6 +132,21 @@ object Analytics {
     reporters.forEach { it.createEvent("DataCleared", props) }
   }
 
+  fun reportArchitectureMigration(
```

Comment:
We should a test for this in `AnalyticsTest`.

Reply:
We really should add a test, because the `hostedScreen` event is not being sent. This is because `.apply` returns the same instance, and not the new instance that was created within the `apply {}` block.

I think we can just create a mutable map instead and just use a simple `if(!hostScreen.isNullOrBlank())` check instead of the current approach.

Reply:
We have a `DebugAnalyticsReporter` that can be added in `DebugClinicApp` to log the analytics events to logcat for verification.

Reply:
Nice catch! Will fix